### PR TITLE
Adapt TIP/TIR to support Infrastructure Components

### PR DESCRIPTION
**Pull request by nichtraunzer**

### CODE REVIEW

**Comment by nichtraunzer** [comment]
fyi @metmajer

**Comment by metmajer** [comment]
This branch is outdated and needs to be updated with the changes in `master` once `master` was updated with the changes made in `release/1.1`. FYI @nichtraunzer

File: templates/TIR-infra.html.tmpl [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+            </li>
+            <li><a href="#section_6">Reference Documents</a></li>
+            <li><a href="#section_7">Document History</a></li>
+        </ol>
+    </div>
+
+    <div class="page">
+        <h2 id="section_1"><span>1</span>Introduction</h2>
+
+        <p>This document describes the installation of the infrastructure component <em>{{metadata.buildParameter.configItem}}</em>. The installation is based on the corresponding Technical Installation Plan (<em>TIP</em>) of Config Item: {{metadata.buildParameter.configItem}}.</p>
```
Comment (martsec) [inline templates/TIR-infra.html.tmpl:94]: I would remove the word 'infrastructure' from here. 
We can use this quickstarter to deploy application components to the cloud, so this word here is a problem. I am thinking about deploying data pipelines for example. And that is not an infrastructure component. 

FYI @metmajer  @frankjoas  @nichtraunzer

**Review by martsec — CHANGES_REQUESTED** [review]

**Comment by metmajer** [comment]
@nichtraunzer @frankjoas I have integrated these changes into https://github.com/opendevstack/ods-document-generation-templates/pull/26. The latter PR is targeted at branch `release/v1.1`. Once the PR has been merged, we'll merge `release/v1.1` into `master` via a dedicated PR and close this PR. You'll be named as contributors ;-)

**Comment by metmajer** [comment]
Merged into `release/v1.1` via https://github.com/opendevstack/ods-document-generation-templates/pull/26 and merged into `master` via https://github.com/opendevstack/ods-document-generation-templates/pull/24.

**Pull request closed without merging**